### DISCUSSION TITLE
Allow setting the comment field of PassthroughDecoder

### DISCUSSION THREAD

**Discussion by irgendwr** [librespot-org/librespot#790] [Feature Requests]
Hi, the new `PassthroughDecoder` is very useful, but in my case it's still missing a feature that I require: access to the `comment` field. My use case is to overwrite the comment header in order to add metadata that my audio player requires.

One way to allow that would be to set the `comment` field to `pub`. If this is not desired for future compatibility reasons, my other suggestion would be to add a simple setter function, such as this:

```rust
pub fn set_comment(&mut self, comment: Box<[u8]>) {
	self.comment = comment;
}
```

Any thoughts on this?

Also: thanks for maintaining such a great client/library! :heart:

**Comment by roderickvd** [comment]
Let's discuss at #791.
